source/MulensModel/magnificationcurve.py

```diff
@@ -366,26 +374,84 @@ def get_binary_lens_magnification(self):
                 Vector of magnifications.
 
         """
-        # Set up the binary lens system
+        out = self._get_binary_lens_magnification(BinaryLens, dict())
+
+        return out
+
+    def get_binary_lens_with_shear_magnification(self):
+        """
+        Calculate magnification for the binary lens with shear and convergence.
+
+        Allowed magnification methods
```

Comment:
If these are always the same as the regular binary lens magnification methods, it would be better to just have a reference to that piece of documentation. Otherwise, it becomes a challenge for long-term maitenance.

Reply:
Good point.
I've thought more about it - maybe we should remove `get_binary_lens_with_shear_magnification()` and `get_binary_lens_magnification()` to return magnification that is for type specified by `parameters` keyword of `__init__()`? Current version can be misleading and if somebody has parameters with shear and wants to get magnification without shear, then this is something non-standard and they should copy parameters to a separate instance of `MagnificationCurve`.

Reply:
Solved in [c8acf3c](https://github.com/rpoleski/MulensModel/pull/15/commits/c8acf3c46161fbc8236cbcdf147caa084d303763) and [89e3cb1](https://github.com/rpoleski/MulensModel/pull/15/commits/89e3cb1fa261c8f59996e2dfd941dbc6dc30d272)